pkg/apis/core/validation/validation.go

```diff
@@ -3408,6 +3408,11 @@ func ValidatePodSecurityContext(securityContext *core.PodSecurityContext, spec *
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("shareProcessNamespace"), *securityContext.ShareProcessNamespace, "ShareProcessNamespace and HostPID cannot both be enabled"))
 			}
 		}
+		if securityContext.HostUserNamespace != nil {
+			if !utilfeature.DefaultFeatureGate.Enabled(features.NodeUserNamespace) {
+				allErrs = append(allErrs, field.Forbidden(fldPath.Child("HostUserNamespace"), "Requesting host user namespace explicitly is disabled by NodeUserNamespace feature-gate"))
```

Comment:
validation error refers to the external field name, so `hostUserNamespace`

Reply:
sure